images/stackrox-build.Dockerfile

```diff
@@ -68,7 +69,7 @@ COPY --from=builder /tmp/rocksdb/include /lib/rocksdb/include
 COPY --from=builder /tmp/rocksdb/ldb /usr/local/bin/ldb
 
 ENV CGO_CFLAGS="-I/lib/rocksdb/include"
-ENV CGO_LDFLAGS="-L/lib/rocksdb -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd"
+ENV CGO_LDFLAGS="-L/lib/rocksdb -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd -ljemalloc"
```

Comment:
lgtm for testing, but what I recently noticed was that this causes _every_ `go build` call where `CGO_ENABLED` is not `0` to be performed with these flags, including `go install` calls that are used to install internal tools (such as `protoc-gen-gofast`). I think that these env vars should be renamed to `ROCKSDB_CGO_CFLAGS` and `ROCKSDB_CGO_LDFLAGS`, and then selectively only set `CGO_FLAGS="${CGO_CFLAGS} ${ROCKSDB_CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS} ${ROCKSDB_CGO_LDFLAGS}" go build ...` when building the main binary.

Reply:
I am going to follow up on this change afterwards